notifikasjon/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/OpprettSakService.kt

```diff
@@ -216,19 +179,49 @@ class OpprettSakService(private val rapidsConnection: RapidsConnection, override
         }
     }
 
-    override fun onError(feil: Fail): Transaction {
-        val utloesendeBehov = Key.BEHOV.lesOrNull(BehovType.serializer(), feil.utloesendeMelding.toMap())
-        if (utloesendeBehov == BehovType.FULLT_NAVN) {
-            val arbeidstakerKey = RedisKey.of(feil.transaksjonId, Key.ARBEIDSTAKER_INFORMASJON)
-            redisStore.set(arbeidstakerKey, ukjentArbeidstaker().toJsonStr(PersonDato.serializer()))
-            return Transaction.IN_PROGRESS
+    private inline fun medTransaksjonIdOgForespoerselId(message: JsonMessage, block: (UUID, UUID) -> Unit) {
```

Comment:
Virker litt omfattende, samt litt voldsomt å duplisere denne metoden  - kunne man heller hatt en felles metode? 
Evt helst en hentTransaksjonId() og en hentForespoerselId() fra message, dette bør være en ganske vanlig ting som gjøres mange flere steder også

Reply:
For transaksjonId så leser man alltid fra melding, men forespoerselId blir av og til lest fra Redis. Da tenker jeg at fremgangsmåten du beskriver blir mer hensiktsmessig ved en senere aneldning. 

Jeg ønsker nemlig å forenkle servicene ved å merge verdier fra innkommende meldinger og fra Redis, slik at man slipper å bry seg om det i koden. Når det er gjort vil en `hentForespoerselId`-funksjon være enklere å ta i bruk.